test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,57 @@
+define([
+  "pentaho/visual/base/View"
+], function(View) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      var data = {};
+      dataSpec = {v: data};
+    });
+
+    describe("the constructor ", function() {
+
+      it("should throw if not invoked with two arguments", function() {
+        expect(function() {
+          return new View();
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element);
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element, dataSpec);
```

Comment:
How else should we test if two arguments are being passed to the constructor? If I pass a nully it's as if I omitted the argument, if I pass a simple {}, I might be sending the wrong message.

The alternative of removing the check from the constructor does not please me.